src/Domain/CoordinateMaps/TimeDependent/RotScaleTrans.hpp

```diff
@@ -39,11 +39,11 @@ namespace domain::CoordinateMaps::TimeDependent {
  * radius and outer radius have a linear radial falloff applied to them.
  * Coordinates at or beyond the outer radius have no translation applied and are
  * expanded by the outer expansion function of time $E_{b}(t)$. This map assumes
- * that the center of the map is at (0., 0., 0.). If translation or expansion is
- * specified, then an inner and outer radius must be specified as well. There is
- * a rigid flag which when set to true treats the coordinates as if they're
- * within the inner radius. This can be used as an optimization if you know
- * which blocks in the domain will always be within the inner radius.
+ * that the center of the map is at (0., 0., 0.). There is an enum class to set
+ * which region your block is in. Specifying Inner treats coordinates as if
+ * they're inside the inner radius, transition treats the points as if they're
+ * between the inner and outer radius. Outer treats points as if they're on or
```

Comment:
Make Inner be `RotScaleTrans::BlockRegion::Inner`. Same for Transition and Outer